bitcoin/tests/psbt.rs

```diff
@@ -0,0 +1,446 @@
+//! Tests PSBT integration vectors from BIP 174
+//! defined at <https://github.com/bitcoin/bips/blob/master/bip-0174.mediawiki#test-vectors>
+
+use std::str::FromStr;
+use std::collections::BTreeMap;
+
+use bitcoin::{Amount, Denomination, Network, OutPoint, PublicKey, PrivateKey, Script, Transaction, Txid, TxOut, TxIn, Witness};
+use bitcoin::consensus::encode::{serialize_hex, deserialize};
+use bitcoin::blockdata::script;
+use bitcoin::blockdata::opcodes::OP_0;
+use bitcoin::hashes::hex::FromHex;
+use bitcoin::secp256k1::{self, Secp256k1};
+use bitcoin::util::bip32::{ExtendedPrivKey, ExtendedPubKey, Fingerprint, IntoDerivationPath, KeySource};
+use bitcoin::util::psbt::{Psbt, PsbtSighashType};
```

Comment:
nit: We are in the process of flattening `util`, would be better if the import path did not use `util` (both `psbt` and `bip32` are in scope at the crate root).